.mkosi/mkosi.debian

```diff
@@ -73,7 +73,9 @@ Packages=
         # We pull in the -dev package here, since the binary ones appear to change names too often, and the -dev package pulls the right deps in automatically
         libtss2-dev
         locales
+        nano
         strace
+        vi
```

Comment:
it's ```vim``` in Debian/Ubuntu

Reply:
The entirety of vim is quite a big bigger than the vi package (at least on Arch) but it doesn't seem like a separate vi is available on Ubuntu/Debian so let's go wirh vim.